docs/domains/Domain.md

```diff
@@ -92,6 +92,7 @@ An element representing a cluster in the domain configuration.
 
 | Name | Type | Description |
 | --- | --- | --- |
+| `distributionStrategy` | string | The strategy for handling changes to the secrets or overridesConfigMap fields. Legal values are DYNAMIC and ROLLING. |
```

Comment:
The thing is, a change to a secret in FromModel is not dynamic at present - custom config overrides are not used in FromModel.